spec/gemfury/app_spec.rb

```diff
@@ -7,11 +7,17 @@ class MyApp < Gemfury::Command::App
       def read_config_file
         { :gemfury_api_key => 'DEADBEEF' }
       end
+    end
 
-      def die!(msg, err = nil, command = nil)
-        raise err
-      end
+    # rspec-mocks re-defines the following methods as part of observing its invocations.
```

Comment:
@rykov as suggested, added a comment, and refactored the code a bit to look cleaner. what do you think?